pkg/redirectpolicy/manager.go

```diff
@@ -492,6 +502,10 @@ func (rpm *Manager) upsertService(config *LRPConfig, frontendMapping *feMapping)
 	}
 }
 
+func (rpm *Manager) checkPodReady(pod *slimcorev1.Pod) bool {
```

Comment:
nit: This method isn't required? Also, why is this added as a method receiver?

Reply:
removed.